source/common/grpc/grpc_web_filter.cc

```diff
@@ -73,6 +72,10 @@ Http::FilterHeadersStatus GrpcWebFilter::decodeHeaders(Http::HeaderMap& headers,
 }
 
 Http::FilterDataStatus GrpcWebFilter::decodeData(Buffer::Instance& data, bool) {
+  if (!is_grpc_web_request_) {
+    return Http::FilterDataStatus::Continue;
```

Comment:
Can we please make sure we have coverage of all of these continue cases? I think it's probably as easy as just calling the other callbacks in your tests and faking a response also for each of headers/data/trailers.